data/channels.csv

```diff
@@ -1331,19 +1331,19 @@ Arroyo.us,Arroyo,,,,US,US-CA,Pasadena,s/US-CA,eng,general,FALSE,,,,https://www.p
 ArroyoGrandeChannel20.us,Arroyo Grande Channel 20,,,,US,US-CA,Arroyo Grande,s/US-CA,eng,,FALSE,,,,https://slo-span.org/channels/arroyo-grande-channel-20,https://i.imgur.com/ZTaEFWY.png
 Arryadia.ma,Arryadia,,,SNRT,MA,,,c/MA,ara,sports,FALSE,2006-09-16,,,https://arryadia.snrt.ma/,https://i.imgur.com/Bckv3Qu.png
 Arsenal.ru,Arsenal,Арсенал,,,RU,,,c/RU,rus,,FALSE,,,,https://arsenal-tv.tv/,https://i.imgur.com/DBpVeb0.png
-ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/OKzWo0r.png
-ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/DqzhSi8.png
+ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/2lKqOjj.png
+ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/st7VS0B.png

```

Comment:
What's wrong with the old logos?

Reply:
The text is in white which appears clearer on darker backgrounds.

Moreover, Ive found better logos from ARTonline.tv official website, should be larger and clearer than the old logos

ARTAflam1.sa
https://i.imgur.com/gPQPjAo.png

ARTAflam2.sa
https://i.imgur.com/q4Hsa4w.png

Reply:
These versions are definitely better.